src/test/resources/db/changelog/db.changelog-master-test.yaml

```diff
@@ -0,0 +1,7 @@
+databaseChangeLog:
+  - include:
+      file: db/changelog/changeset/init/01-init-problem-test.yaml
+  - include:
+      file: db/changelog/changeset/init/02-init-problem-tags-test.yaml
+  - include:
```

Comment:
It will be executed each time on every test. It's better to prepare required data separately and independently for each test and avoid scenario when all tests affected.

Reply:
It's okay to have DDL as a common part, DML should be separated.

Reply:
Fixed